CHANGES_UNRELEASED.md

```diff
@@ -3,3 +3,7 @@
 # Unreleased Changes
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.58.2...master)
+
+## FxA Client
+- iOS: Added optional boolean argument `forceRefresh` to ignore caching for `fetchDevices`. ([#3066](https://github.com/mozilla/application-services/pull/3066))
+- Android: Added option boolean argument `ignoreCache` to ignore caching for `getDevices`. ([#3066](https://github.com/mozilla/application-services/pull/3066))
```

Comment:
Oh wow it's even worse when you see the two different names next to each other in the changelog like this 😆 

I will defer to @eoger on the wisdom or safety of attempting to rename the iOS version for consistency.

Reply:
FWIW `CHANGES_UNRELEASED.md` is empty at the moment and needs to be rebased :)
But yeah I support this breaking change, iOS only uses the `forceRefresh` argument [in 1 place](https://searchfox.org/mozilla-mobile/rev/fd32c1b030795f42f68da92c36e9bdef97b52d29/firefox-ios/RustFxA/FxAWebView.swift#163) so it won't be too bad.